style.css

```diff
@@ -169,4 +160,283 @@
     order: 3;
     flex-grow: 0;
   }
+
+  /* from here */
+  .snapshot1 {
+    margin-top: 30px;
+    height: 570px;
+    width: 375px;
+    border-radius: 16px;
+    padding: 16px;
+    background: white;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12px;
+  }
+
+  body {
+    background: #e5e5e5;
+  }
+
+  .head-one {
+    /* Project Title */
+    width: 295px;
+    height: 20px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: -20px;
+
+    /* Neutrals/N800 */
+    color: #172b4d;
+  }
+
+  .doc {
+    width: 295px;
+    height: 76px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+
+    /* Neutrals/N600 */
+    color: #344563;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  ul li {
+    text-decoration: none;
+  }
+
+  .grid-container {
+    margin-top: 30px;
+    display: grid;
+    grid-template-columns: auto auto auto;
+    height: 32px;
+    min-width: 310px;
+    align-items: center;
+  }
+
+  .grid-item {
+    display: flex;
+    justify-content: space-between;
+    font-size: 20px;
+    padding: 5px;
+    order: 1;
+  }
+
+  .counter {
+    height: 10px;
+    margin-top: 8px;
+    margin-right: 5px;
+  }
+
+  .tags {
+    height: 24px;
+    width: 224px;
+    top: 200px;
+  }
+
+  .canop {
+    color: #020303;
+  }
+
+  #grey-works {
+    color: #7a869a;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    justify-content: space-between;
+    column-gap: 10px;
+  }
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    font-size: 14px;
+  }
+
+  a:link {
+    text-decoration: none;
+    color: #6070ff;
+  }
+
+  .btn {
+    /* Tag */
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .action {
+    height: 72px;
+    width: 295px;
+    margin-top: 35px;
+  }
+
+  .normal-btn {
+    margin-left: 10px;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    position: absolute;
+    width: 124px;
+    height: 48px;
+
+    /* Primary/B400 */
+
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+  }
+
+  .btn-text {
+    color: #396df2;
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    font-style: normal;
+  }
```

Comment:
- [ ] Please implement different styles of your buttons based on various user interactions like `hover`, `active`, `pressed`, etc. This enhances user experience and makes the website awesome 😉 You have been provided with the design guidelines to follow 👇 
![image](https://user-images.githubusercontent.com/90222110/192117935-2e8d48fc-d3a6-40f5-b5e4-987383265dec.png)